plugins/scanners/scancode/src/main/kotlin/ScanCode.kt

```diff
@@ -90,18 +90,15 @@ class ScanCode internal constructor(
 
     override val writeToStorage by lazy { wrapperConfig.writeToStorageWithDefault(matcher) }
 
-    override val configuration by lazy {
-        buildList {
-            addAll(config.commandLine)
-            add(OUTPUT_FORMAT_OPTION)
-        }.joinToString(" ")
-    }
+    override val configuration by lazy { config.commandLine.joinToString(" ") }
 
     internal fun getCommandLineOptions(version: String) =
         buildList {
             addAll(config.commandLine)
             addAll(config.commandLineNonConfig)
```

Comment:
I do not understand why output format does not become part of `commandLineNonConfig`. Can you help me there?

Reply:
Good point, it should be, but it was more difficult before as `OUTPUT_FORMAT_OPTION` was private to `ScanCode`. Will add another commit after the inlining of that variable to do that, which avoid several rebasing conflicts for me.